packages/composite-checkout/src/components/checkout-payment-methods.js

```diff
@@ -11,19 +11,24 @@ import styled from '@emotion/styled';
 import joinClasses from '../lib/join-classes';
 import RadioButton from './radio-button';
 import { useLocalize } from '../lib/localize';
-import { useAllPaymentMethods } from '../public-api';
+import {
+	useAllPaymentMethods,
+	usePaymentMethod,
+	usePaymentMethodId,
+	useActiveStep,
+} from '../public-api';
 import CheckoutErrorBoundary from './checkout-error-boundary';
 
 export default function CheckoutPaymentMethods( {
 	summary,
 	isComplete,
 	className,
 	availablePaymentMethods,
-	paymentMethod,
-	onChange,
 } ) {
 	const localize = useLocalize();
 
+	const paymentMethod = usePaymentMethod();
+	const [ , setPaymentMethod ] = usePaymentMethodId();
```

Comment:
Just clarifying - the notation here roughly means "we don't care about the first argument of this array, so don't bother allocating a name for it", right?

Reply:
Correct.